torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15680,16 +15686,9 @@ def ref_pairwise_distance(input1, input2):
     ),
     OpInfo(
         'scatter_reduce',
-        dtypes=all_types_and(torch.float16, torch.bfloat16),
```

Comment:
How come this was working beforehand?

Reply:
Previously this was enabled for cpu only, I sent a longer discussion regarding this offline